gcloud/bigtable/test_cluster.py

```diff
@@ -53,6 +53,30 @@ def test_constructor_non_default(self):
         self.assertEqual(cluster.serve_nodes, serve_nodes)
         self.assertTrue(cluster._client is client)
 
+    def test_copy(self):
+        project = 'PROJECT'
+        zone = 'zone'
+        cluster_id = 'cluster-id'
+        display_name = 'display_name'
+        serve_nodes = 8
+
+        client = _Client(project)
+        cluster = self._makeOne(zone, cluster_id, client,
+                                display_name=display_name,
+                                serve_nodes=serve_nodes)
+        new_cluster = cluster.copy()
+
+        # Make sure the client got copied to a new instance.
+        self.assertFalse(new_cluster._client is client)
+        self.assertEqual(new_cluster._client.__dict__,
+                         client.__dict__)
+
+        # Just replace the client on the new_cluster so we can
```

Comment:
The `__eq__` testing for the `Cluster` class was done while bringing over `Client` (since the `list_clusters` unit tests depended on `Cluster.__eq__` working).

I'm not sure what else you might be asking. Not checking `_client.__dict__`? I believe I did that to avoid implementing `__eq__` on the `_Client` helper, but I'm only 95% sure.